server/routes/reviews.js

```diff
@@ -0,0 +1,77 @@
+const router = require("express").Router();
+const { User, Product, Review } = require("../db/index.js");
+
+//All reviews
```

Comment:
Why would we ever send all reviews ever made to the client? This route feels confusing to me...